astropy/cosmology/io/html.py

```diff
@@ -0,0 +1,140 @@
+import astropy.cosmology.units as cu
+import astropy.units as u
+from astropy.cosmology.connect import readwrite_registry
+from astropy.cosmology.core import Cosmology
+from astropy.cosmology.parameter import Parameter
+from astropy.table import QTable
+
+from .table import from_table, to_table
+
+# format look-up for conversion, {original_name:new_name}
+format_table = {"H0": "$$H0$$",
+                "Om0": "$$Om0$$",
+                "Ode0": "$$Ode0$$",
+                "Tcmb0": "$$Tcmb0$$",
+                "Neff": "$$Neff$$",
+                "m_nu": "$$m_{nu}$$",
+                "Ob0": "$$Ob0$$",
+                "w0": "$$w0$$",
+                "wa": "$$wa$$",
+                "wz": "$$wz$$",
+                "wp": "$$wp$$",
+                "zp": "$$zp$$"}

```

Comment:
A good start, but let's use the cosmological symbols

```suggestion
format_table = {"H0": "$$H_{0}$$",
                "Om0": "$$\Omega_{m,0}$$",
                "Ode0": "$$\Omega_{\Lambda,0}$$",
                "Tcmb0": "$$T_{0}$$",
                "Neff": "$$N_{eff}$$",
                "m_nu": "$$m_{nu}$$",
                "Ob0": "$$\Omega_{b,0}$$",
                "w0": "$$w_{0}$$",
                "wa": "$$w_{a}$$",
                "wz": "$$w_{z}$$",
                "wp": "$$w_{p}$$",
                "zp": "$$z_{p}$$"}
```